protocol/testutil/constants/clob_pair.go

```diff
@@ -127,4 +127,40 @@ var (
 		QuantumConversionExponent: -8,
 		Status:                    clobtypes.ClobPair_STATUS_PAUSED,
 	}
+	ClobPair_Iso = clobtypes.ClobPair{
```

Comment:
Nit: `ClobPairId_3_Iso`, same with below. `ClobPair_Iso2` has Id 4 so it gets a little confusing during tests, maybe ClobPairId_4_Iso